tests.md

```diff
@@ -0,0 +1,228 @@
+export WPT_TEST_DIR=/tmp/wordpress
+
+# API key to authenticate with the reporting service in 'username:password' format.
+export WPT_REPORT_API_KEY=
+
+# (Optionally) define an alternate reporting URL
+export WPT_REPORT_URL=
+
+# Credentials for a database that can be written to and reset. WARNING!!! This database will be destroyed between tests. Only use safe database credentials.
+export WPT_DB_NAME=wordpress
+export WPT_DB_USER=wordpress
+export WPT_DB_PASSWORD=__PASSWORD__
+export WPT_DB_HOST=localhost
+
+# (Optionally) set a custom table prefix to permit concurrency against the same database.
+export WPT_TABLE_PREFIX=${WPT_TABLE_PREFIX-wptests_}
+
+# (Optionally) define the PHP executable to be called
+export WPT_PHP_EXECUTABLE=${WPT_PHP_EXECUTABLE-php}
+
+# (Optionally) define the PHPUnit command execution call. Use if `php phpunit.phar` can't be called directly for some reason.
+export WPT_PHPUNIT_CMD=
+
+# (Optionally) define the command execution to remove the test directory. Use if `rm -r` can't be called directly for some reason.
+export WPT_RM_TEST_DIR_CMD=
+
+# SSH connection string (can also be an alias). Leave empty if tests are meant to run in the same environment.
+export WPT_SSH_CONNECT=
+
+# Any options to be passed to the SSH connection. Defaults to '-o StrictHostKeyChecking=no'
+export WPT_SSH_OPTIONS=
+
+# SSH private key, base64 encoded.
+export WPT_SSH_PRIVATE_KEY_BASE64=
+
+# Output logging. Use 'verbose' to increase verbosity
+export WPT_DEBUG=
+```
+
+We will configure the folder where the WordPress software downloads and the database accesses will be made in order to prepare the tests.
+
+### Preparing the environment
+
+Before we do the first test, we'll catch up on everything. This process can be run before each test in this environment if we want to keep it up to date, although it will depend more if it is in a production environment.
+
+```bash
+cd /tmp/phpunit-test-runner/
+git pull
+npm update
+source .env
+```
+
+### Preparing the test
+
+Now that we have the environment ready, we can run the test preparation.
+
+```bash
+cd /tmp/phpunit-test-runner/
+php prepare.php
+```
+
+The system will run a long series of installations, configurations and compilations of different elements in order to prepare the test. If warnings and warnings come out you should not worry too much, as it is quite normal. At the end of the process it will warn you if it needs something it doesn't have. In principle, there should be something like this:
+
+```
+Done.
+Replacing variables in wp-tests-config.php
+Success: Prepared environment.
+```
+
+If the environment has been prepared, the next step is to make a first test.
+
+### Running the test
+
+Now that we have the environment ready, let's go get the test. To do this we will execute the file that will perform it.
```

Comment:
"Now that the environment is ready, let's run the tests."

Reply:
Updated